templates/repo/actions/runs_list.tmpl

```diff
@@ -24,6 +24,12 @@
 						{{ctx.Locale.Tr "actions.runs.pushed_by"}}
 						<a href="{{$run.TriggerUser.HomeLink}}">{{$run.TriggerUser.GetDisplayName}}</a>
 					{{- end -}}
+					{{$errMsg := index $.RunErrors $run.ID}}
+					{{if $errMsg}}
+						<span data-tooltip-content="{{$errMsg}}">
+							{{svg "octicon-alert" 16 "text red"}}
+						</span>
+					{{end}}
```

Comment:
The icon is slightly off-center vertically. I think we can potentially fix it by adding `flex-text-block` to the `flex-item-body` element. The icon in the sidebar would need similar treatment as per your screenshot.

Reply:
Looks better, but still like 1px too high, but I guess that's acceptable.

One last request: Can you add like `tw-ml-2` to the elements so they have a bit more space from the text?

Reply:
No `tw-ml-2`, correct `flex-text-block` should be able to correctly add the gap